### PR TITLE
cluster: ux improvements

### PR DESCRIPTION
**Pull request by tamarrow**
* open DC/OS clusters don't have a cluster cert, default to insecure requests
* add --attached param to `cluster list`
* improve error messages
* use current config for http._verify_ssl

### CODE REVIEW

File: cli/dcoscli/data/help/cluster.txt [hunk 1]
```diff
@@ -28,6 +28,8 @@ Commands:
         Setup the CLI to talk to your DC/OS cluster.
 
 Options:
+    --attached
+        List only attached cluster.
```
Comment (jgehrcke) [inline cli/dcoscli/data/help/cluster.txt]: "List only the currently attached cluster"?

File: cli/dcoscli/data/help/cluster.txt [hunk 2]
```diff
@@ -37,7 +39,7 @@ Options:
     --insecure
         Allow requests to bypass SSL certificate verification (insecure).
     --no-check
-        Do not check CA certficate downloaded from cluster (insecure).
+        Do not check CA certficate downloaded from cluster (insecure). Enterprise only.
```
Comment (jgehrcke) [inline cli/dcoscli/data/help/cluster.txt]: "Applies to Enterprise DC/OS only."?

File: dcos/cluster.py [hunk 3]
```diff
@@ -182,8 +182,13 @@ def get_cluster_cert(dcos_url):
         with urlopen(cert_bundle_url, context=unverified) as f:
             return f.read().decode('utf-8')
     except Exception as e:
-        msg = "Error downloading CA cert from cluster"
-        raise DCOSException("{}:\n{}".format(msg, e))
+        # DC/OS open clusters do not have certs
```
Comment (jgehrcke) [inline dcos/cluster.py]: Maybe "Open source DC/OS does not currently expose its root CA certificate"

File: dcos/cluster.py [hunk 4]
```diff
@@ -182,8 +182,13 @@ def get_cluster_cert(dcos_url):
...
+        if e.code == 404:
```
Comment (jgehrcke) [inline dcos/cluster.py]: if the exception here does not inherit from requests' HTTPError then `e` may not have the `code` property, resulting in an AttributeError. A TCP layer problem might cause this (ConnectionRefusedError etc).

Comment (tamarrow) [inline dcos/cluster.py]: Good catch.

**Pull request merged**